package.json

```diff
@@ -17,7 +17,10 @@
     "next": "13.2.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "sass": "^1.59.2",
+    "react-icons": "^4.8.0",
     "typescript": "4.9.5"
+  },
+  "devDependencies": {
```

Comment:
Remove it as dev-dependency, and install it back as normal dependency. To do that:
1. `npm uninstall sass`
2. `npm i sass`

Reply:
Removed sass as dev-dependency and added as normal dependency.

Reply:
@FaizBShah I was curious why choose to install it as normal dependency. It is required only for development purposes and no need to install this in production right?

Reply:
@BIKRAM-SAHA You're correct. I don't remember why but I think it was to keep the packages consistent since I was experimenting with the boilerplate, and didn't wanted to get errors and waste time figuring out which dependencies can work in dev-mode and which can work in in prod mode. But now I think barring 4 (next, react, react-dom, react-router-dom), all of them could be installed as dev-dependency, as none of them will be actually required in the production build. (If we actually required typescript in production, my life would had been so much better :p)